lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -9,7 +9,7 @@ class PokemonOverviewPage extends StatelessWidget {
     required this.pokemonList,
   });
 
-  final List<Pokemon> pokemonList;
+  final List<PokemonDto> pokemonList;
```

Comment:
kindly have a type_def for this 🙂 

create a `typedefs.dart` class in your util directory then => `typedef PokemonList = List<PokemonDto>;`